### PR TITLE
Remove text color from notification mails and fix sender avatar

### PR DESCRIPTION
**Pull request by SuperTux88**
I use a dark desktop theme, and gray text on gray background is not really readable ;)

Removing the text color fixes this problem, because the mail client uses the default text color then (which is white on a dark theme and black on a bright theme). It already uses the default background-color of the theme.

And I saw that there is broken sender-avatar code in the mail template, and fixed it. So this PR brings also back the sender avatar in notification mails.

### CODE REVIEW

File: app/views/layouts/notifier.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+%table{cellspacing: 0, cellpadding: 0, border: 0, style: "font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-size: 14px"}
```
Comment (diaspora-code-review) [inline app/views/layouts/notifier.html.haml]: Line is too long. [136/120]

File: app/views/layouts/notifier.html.haml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  %tr
+    - if @notification.try(:sender)
+      %td{width: "60px", style: "vertical-align: top"}>
+        %div{style: "background-color: #eee; height: 50px; width: 50px"}
+          = image_tag @notification.sender.profile.image_url(:thumb_small), alt: @notification.sender.name, style: "border: 0, display: block; top: 0; left: 0", height: "50px", width: "50px"
```
Comment (diaspora-code-review) [inline app/views/layouts/notifier.html.haml]: Line is too long. [190/120]

File: app/views/layouts/notifier.html.haml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    %td{style: "vertical-align: top"}
+      != yield
+
+%div{style: "font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-size: 12px; color: #999; padding-top: 10px; margin-top: 10px; border-top: 1px solid #ddd"}
```
Comment (diaspora-code-review) [inline app/views/layouts/notifier.html.haml]: Line is too long. [169/120]

File: app/views/layouts/notifier.html.haml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  %tr
+    - if @notification.try(:sender)
+      %td{width: "60px", style: "vertical-align: top"}>
+        %div{style: "background-color: #eee; height: 50px; width: 50px"}
+          = image_tag @notification.sender.profile.image_url(:thumb_small), alt: @notification.sender.name,
+                      style: "border: 0, display: block; top: 0; left: 0", height: "50px", width: "50px"
+    %td{style: "vertical-align: top"}
+      != yield
+
+%div{style: "font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-size: 12px; color: #999; padding-top: 10px; margin-top: 10px; border-top: 1px solid #ddd"}
```
Comment (diaspora-code-review) [inline app/views/layouts/notifier.html.haml]: Line is too long. [169/120]

**Comment by svbergerem** [comment]
Merged as 5017bd9541f30be10c21e76185bfe1cba47742ec b0611ce2701afe9adb8e52966f3beab5a62365a4 364502a134b30a7e1af7625c5ab528da166b2708 575748169ec1d580695a7e397e36eaf6c2bc548c. Thanks! :)

**Pull request closed without merging**